PWG/EMCAL/EMCALbase/AliAnalysisTaskEmcalEmbeddingHelper.cxx

```diff
@@ -615,6 +623,83 @@ bool AliAnalysisTaskEmcalEmbeddingHelper::IsRunInRunlist(const std::string & pat
   return false;
 }
 
+/**
+ * Get the data background run number from the data file path, fRunNumberBgk and 
+ * Determine the MC run block range,  fEmbeddedRunblockMin and  fEmbeddedRunblockMax (used in IsRunInRunblock())
+ * Right now only done once, for the first data file
+ */
+void AliAnalysisTaskEmcalEmbeddingHelper::SetRunblockRange() 
+{
+  Int_t nBlocks = fEmbeddedRunblock.size();
+  
+  if ( nBlocks == 0 ) return ;
+  
+  AliDebug(1,Form("AliAnalysisTaskEmcalEmbeddingHelper::SetRunblockRange() - nRunBlocks %d\n", nBlocks));
```

Comment:
Please use `AliDebugStream(1) << "nRunBlocks " << nBlocks << "\n";` No doubt it's not always used everywhere, but I'd like to try to maintain it when possible

Reply:
OK